modules/run-vault/run-vault

```diff
@@ -112,16 +115,34 @@ function generate_vault_config {
   local readonly api_addr="$5"
   local readonly config_dir="$6"
   local readonly user="$7"
-  local readonly enable_s3_backend="$8"
-  local readonly s3_bucket="$9"
-  local readonly s3_bucket_region="${10}"
+
+  if [[ "$enable_s3_backend" == "true" ]]; then
+    local readonly enable_s3_backend="$8"
+    local readonly s3_bucket="$9"
+    local readonly s3_bucket_region="${10}"
+    local readonly enable_dynamo=""
+    local readonly dynamo_region=""
+    local readonly dynamo_table=""
+  fi
+
+  if [[ "$enable_dynamo" == "true" ]]; then
+    local readonly enable_dynamo="$8"
+    local readonly dynamo_region="$9"
+    local readonly dynamo_table="${10}"
+    local readonly enable_s3_backend=""
+    local readonly s3_bucket=""
+    local readonly s3_bucket_region=""
+  fi
+
   local readonly config_path="$config_dir/$VAULT_CONFIG_FILE"
 
   local instance_ip_address
   instance_ip_address=$(get_instance_ip_address)
 
   log_info "Creating default Vault config file in $config_path"
   local readonly listener_config=$(cat <<EOF
+ui = true
```

Comment:
Enabled vault UI by default

Reply:
Thx! We should probably make this configurable though with a `--disable-ui` flag for those that want to leave it off.